config/zones.json

```diff
@@ -383,12 +383,21 @@
   },
   "CA-QC": {
     "_comment": "http://www.hydroquebec.com/generation/",
+    "_comment": "http://www.hydroquebec.com/decom",
     "capacity": {
       "gas": 411,
-      "hydro": 36366,
-      "oil": 130
+      "hydro": 41859,
+      "oil": 130,
+      "wind": 3508,
+      "unknown": 1056,
+      "nuclear": 657
     },
-    "flag_file_name": "ca.png"
+    "flag_file_name": "ca.png",
+    "contributors": [
+      "https://github.com/jarek",
+      "https://github.com/corradio",
```

Comment:
No need to add me (I'm merely reviewing)

Reply:
removed